### ISSUE TITLE
Proposal: Notification when losing rank #1 on a beatmap

### ISSUE THREAD

**Issue by mrowswares** [ppy/osu-web#7885]
it appears in the profile recent section so would be cool to get notifications about it
![image](https://user-images.githubusercontent.com/83023433/126043204-98dfdc9d-8b09-478f-83a1-24c7034e8aaf.png)

toggleable in settings i imagine

**Comment by peppy** [comment]
I don't think we have an API to trigger arbitrary notifications, but this seems like one cool thing to add once we move score processing across to newer systems.
